api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -156,15 +156,16 @@ public String toString() {
   private int recursionCount;
 
   /**
-   * Handles newly resolved server groups and metadata attributes from name resolution system.
-   * {@code servers} contained in {@link EquivalentAddressGroup} should be considered equivalent
-   * but may be flattened into a single list if needed.
+   * Handles newly resolved addresses and metadata attributes from name resolution system.
+   * {@link EquivalentAddressGroup} addresses should be considered equivalent but may be flattened
```

Comment:
"Addresses in {@link EquivalentAddressGroup}"? It isn't clear what EAG addresses is talking about; e.g., are EAG addresses a type of address?

Reply:
@code servers is removed from the existing javadoc. Can you elaborate what is expected here?

Reply:
Use "Addresses in {@link EquivalentAddressGroup}" instead of "{@link EquivalentAddressGroup}"

Reply:
Use "Addresses in {@link EquivalentAddressGroup}" instead of "{@link EquivalentAddressGroup} addresses"!

Reply:
Addressed as mentioned